src/app/controller/services/mapping/timestamp-mapping.service.ts

```diff
@@ -126,12 +129,18 @@ export class TimestampMappingService {
 
 
   getLocationNameOptionLabel(timestampType: TimestampDefinitionTO): string {
-    if (timestampType?.isBerthLocationNeeded) {
+    if (timestampType.eventLocationRequirement == EventLocationRequirement.OPTIONAL || 
+      timestampType.eventLocationRequirement == EventLocationRequirement.REQUIRED) {    
+    if (timestampType?.facilityTypeCode == FacilityTypeCode.BRTH) {
       return this.locationNameBerth;
     }
-    if (timestampType?.isPBPLocationNeeded) {
+    if (timestampType?.facilityTypeCode == FacilityTypeCode.PBPL) {
       return this.locationNamePBP;
+    }  
+    if (timestampType?.facilityTypeCode == FacilityTypeCode.ANCH) {
+      return this.locationNameAnchorage;
     }
+  }
     return undefined;
```

Comment:
Indentation looks very weird (at least on github).  Can you double check the indentation is proper?

Reply:
Concretely what I am seeing is: ![image](https://user-images.githubusercontent.com/70640050/183636710-807bc626-9d47-4fd4-8211-c7bb9a4bb3db.png)

Reply:
I will look into it today (y)